models/zeroex/optimism/zeroex_optimism_api_fills.sql

```diff
@@ -150,168 +152,61 @@ ERC20BridgeTransfer AS (
             INDEX                                   AS evt_index,
             logs.contract_address,
             block_time                              AS block_time,
-            '0x' || substring(DATA, 283, 40)        AS maker,
-            '0x' || substring(DATA, 347, 40)        AS taker,
-            '0x' || substring(DATA, 27, 40)         AS taker_token,
-            '0x' || substring(DATA, 91, 40)         AS maker_token,
-            bytea2numeric(substring(DATA, 155, 40)) AS taker_token_amount_raw,
-            bytea2numeric(substring(DATA, 219, 40)) AS maker_token_amount_raw,
+             bytearray_substring(DATA, 142, 20) AS maker,
+            bytearray_substring(DATA, 172, 20) AS taker,
+            bytearray_substring(DATA, 14, 20) AS taker_token,
+            bytearray_substring(DATA, 45, 20) AS maker_token,
+            bytearray_to_uint256(bytearray_substring(DATA, 77, 20)) AS taker_token_amount_raw,
+            bytearray_to_uint256(bytearray_substring(DATA, 110, 20)) AS maker_token_amount_raw,
             'ERC20BridgeTransfer'                   AS type,
             zeroex_tx.affiliate_address             AS affiliate_address,
             TRUE                                    AS swap_flag,
             FALSE                                   AS matcha_limit_order_flag
     FROM {{ source('optimism', 'logs') }} logs
     INNER JOIN zeroex_tx ON zeroex_tx.tx_hash = logs.tx_hash
-    WHERE topic1 = '0x349fc08071558d8e3aa92dec9396e4e9f2dfecd6bb9065759d1932e7da43b8a9'
+    WHERE topic0 = 0x349fc08071558d8e3aa92dec9396e4e9f2dfecd6bb9065759d1932e7da43b8a9
```

Comment:
`topic1` in spark sql becomes `topic0` in dune sql.
https://dune.com/docs/migrations/SparkSQL/#migrating-from-sparksql